frontend/src/partners/PartnerForm.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react'
+import { Formik, Form, Field } from 'formik'
+import { Link } from 'react-router-dom'
+import Input from '../Input'
+import Button from '../Button'
+import PopUp from '../PopUp'
+import schema from './ValidationPartner'
+import RadioInputGroup from '../RadioInputGroup'
+import RadioInput from '../RadioInput'
+
+function PartnerForm({
+  sent,
+  handleClick,
+  popUpMessage,
+  sendData,
+  title,
+  partner,
+}) {
+  return (
+    <div className="container my-5">
+      <div className="row justify-content-center">
+        <div className="col-12">
+          {sent && <PopUp handleClick={handleClick} body={popUpMessage} />}
+          <Formik
+            initialValues={{
+              partnerEmail: '',
```

Comment:
Ezeket az elnevezéseket magyarul szeretnénk használni a kódban?

Reply:
Igen, ezt backenden is így kérték.